tests/unit/test_routes.py

```diff
@@ -979,6 +1108,12 @@ def test_method_not_allowed_handler(module_client):
 # Other Routes
 ##########################################
 
+def test_health_check(module_client):
+    response = module_client.get('/healthz')
+    assert (response.status_code == 200)
+    print(response.get_json())
+    assert (response.get_json().get("application").get("status") == "ok")
```

Comment:
I decided to add this in just because I was in the tests and I noticed we didn't have a test for it. It has nothing to do with this PR but I hope that's ok